File: easy/lengthoflastword.cpp
#include <iostream>

class Solution {
public:
    int lengthOfLastWord(string s) {
        int len = 0;
        int size = s.length() - 1;

        for(int i = size; i >= 0; i--) {
            if(s[i] == ' ') {
                size -= 1;
            } else {
                break;
            }
        }

        for(int i = size; i >= 0; i--) {
            if(s[i] != ' ') {
                len += 1;
            } else {
                break;
            }
        }

        return len;
    }
};

File: python/easy/lengthoflastword.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        length = 0
        size = len(s) - 1

        for i in range(size, -1, -1):
            if s[i] == ' ':
                size -= 1
            else:
                break

        for i in range(size, -1, -1):
            if s[i] != ' ':
                length += 1
            else:
                break

        return length